TileStache/Goodies/VecTiles/sort.py

```diff
@@ -76,8 +77,16 @@ def landuse(features, zoom):
     return _sort_by_area_then_id(features)
 
 
+def _place_key_desc(feature):
+    sort_key = _by_population(feature), _by_area(feature)
```

Comment:
Do neighborhoods have population? Or is this generally for any "place" feature with population?